### ISSUE TITLE
Use static files from /dist folder

### ISSUE THREAD

**Issue by ajayns** [parcel-bundler/parcel#4561]
## 🤔 Expected Behavior

A means to import static files that need no processing. Maybe an output dir variable (`__outputDir`) which Parcel converts to new relative paths during compilation.

## 💁 Possible Solution

For example, if I place say an image in `dist/images/test.jpg`, I should be able to use that in my `index.html` as something like `<img src="__outputDir/images/test.jpg" />

## 🔦 Context

This could help in cases where Parcel needn't manually copy many files (such as images, icons etc) for larger projects, greatly reducing build time and effort.

## 💻 Examples

Webpack has similar variables I believe: https://webpack.js.org/guides/public-path/

**Comment by mischnic** [comment]
> This could help in cases where Parcel needn't manually copy many files

In the end, you still have to "manually" copy them. So I don't see what advantage this would have.

This could be implemented using a named pipeline in Parcel 2 (e.g. `raw: images/test.jpg`).

**Comment by ajayns** [comment]
Say there are about 100 large image files in the project which won't be updated later on after adding to the project. In such a case, doesn't this option reduce the load for copying files each time (I'm assuming Parcel copies all these files each time I run it and that contents of `dist` folder isn't deleted on each run) and this also prevents bloating of the project directory by eliminating 2 copies of the same file?

**Comment by speigg** [comment]
Right, static resources could be massive, such as 3D models and textures. Copying them is not a solution.

**Comment by 001101** [comment]
isn't this not already fixed in this repo since 2018, but still not merged?

https://github.com/naturalethic/parcel/commit/c99a37a0577c9d7a48fc3b68168c3120396bb289

https://github.com/parcel-bundler/parcel/issues/303

**Comment by t3hmrman** [comment]
For those who might run into this in the future and want a simple solution that works relatively easily and as expected for Parcel 2 [`elwin013/parcel-reporter-static-files-copy`](https://github.com/elwin013/parcel-reporter-static-files-copy)
